website/src/components/Messages/RenderedMarkdown.tsx

```diff
@@ -1,3 +1,4 @@
+import { CopyIcon, CheckIcon } from "@chakra-ui/icons";
```

Comment:
Can you update to use https://lucide.dev/ icon? and remove the `@charka-ui/icons` deps

Reply:
Should I use `clipboard-copy` or `clipboard`?

Reply:
You can use copy which is like Github does